app/pages/ServiceListingPage.jsx

```diff
@@ -23,7 +23,14 @@ import { getSiteTitle } from '../utils/whitelabel';
 
 // TODO This should be serviceAtLocation
 const getServiceLocations = (service, resource, recurringSchedule) => {
-  const { addresses = [] } = resource;
+  let addresses;
+  if (service.addresses && service.addresses.length > 0) {
```

Comment:
nit: couldn't we use optional chaining here? `service.addresses?.length > 0` ?

Reply:
Unfortunately, our Babel version is too old to recognize the optional chaining syntax, so we'd need to upgrade that first. But yes, it would be ideal to use that in a situation like this, so good eye!